src/html5_playback.js

```diff
@@ -79,10 +79,20 @@ export default class HTML5TVsPlayback extends Playback {
   }
 
   _setupSource(sourceURL) {
```

Comment:
We have changed the `_setupSource` method to use the `source` element but we did not update the `_wipeUpMedia` as well. According to the [standard's best practice](https://html.spec.whatwg.org/multipage/media.html#best-practices-for-authors-using-media-elements) it is recommended to remove all `source` descendants from the `video` element in order to do the full clean up.

Reply:
Never mind. I see that you did that in this [PR](https://github.com/clappr/clappr-html5-tvs-playback/pull/3)